webui/react/src/components/Spinner.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import Spinner from './Spinner';
+
+describe('Spinner', () => {
+  loadAntdStyleSheet(); // defined in setupTests.ts
```

Comment:
I considered that, but it seems to go against [don't test implementation details.](https://determinedai.atlassian.net/wiki/spaces/ENGINEERIN/pages/1056440325/Automated+Testing#Does-this-test-actually-test-expected-UI-behavior-or-does-it-test-the-implementation-detail%3F)
> What we want to avoid is writing tests for implementation details, but rather to simulate the actions the user would perform with the UI and checking for what the user would expect the outcome to be based on the actions.

I think I'm going to convert this to a draft for now, pending a decision about testing styling dependent interactions. 

Thanks for the review, I will make the other changes.